### DISCUSSION TITLE
How do I display a `tree` in `vitePress`?

### DISCUSSION THREAD

**Discussion by mezhanglei** [vuejs/vitepress#3721] [Q&A]
I want to display a tree conveniently in `vitePress`, but have no ideas

**Comment by zhangfisher** [comment]
recommend use `litetree`

[github](https://github.com/zhangfisher/lite-tree)

```md
<LiteTree>
- A公司
    行政中心
        总裁办
        人力资源部
        财务部
        行政部
        法务部
        审计部
        信息中心
    + 市场中心
        市场部
        销售部
        客服部
        品牌部
        市场策划部
        市场营销部
    研发中心
        移动研发部
        平台研发部
        测试部
        运维部
        产品部
        设计部
</LiteTree>

```

**output:**

![image](https://github.com/vuejs/vitepress/assets/2050579/a4b49ae8-e99a-439b-b128-db7934700bba)

**Reply by fjun99** [reply]
这个超棒。加在 一个 vitepress 教程里面了： https://www.alang.ai/vitepress/v09.html

教程首页：https://www.alang.ai/vitepress/

**Comment by zhangfisher** [comment]
In the component slot, using a YAML like approach to represent indentation to describe trees is a very good experience,:)

`LiteTree` also supports a variety of styles, specifically designed for 'vitepress'

![image](https://github.com/vuejs/vitepress/assets/2050579/511054f5-cb1f-435d-b529-32fba5a7136a)
